website/docs/language/checks/index.mdx

```diff
@@ -0,0 +1,114 @@
+
+[Learn more about assertions](/terraform/language/expressions/custom-conditions#check-assertions).
+
+### Meta-Arguments
+
+Check blocks do not currently support [meta-arguments](/terraform/language/resources/syntax#meta-arguments). We are still collecting feedback on this feature, so if your use case would benefit from check blocks supporting meta-arguments, please [let us know](https://github.com/hashicorp/terraform/issues/new/choose).
+
+## Continuous Validation in Terraform Cloud
+
+Terraform Cloud can automatically validate whether checks in a workspace’s configuration continue to pass after Terraform provisions new infrastructure. See [Continuous Validation](/terraform/cloud-docs/workspaces/health#continuous-validation) for details.
+
+## Choosing Checks or other Custom Conditions
+
+Check blocks offer the most flexible validation solution within Terraform. As you can reference outputs, variables, resources and data sources within check assertions, they can be used to model all the alternate [Custom Conditions](/terraform/language/expressions/custom-conditions). However, that does not mean you should replace all your custom conditions with check blocks.
+
+As discussed, the major behavioral difference between [check block assertions](#assertions) and other custom conditions is that they do not affect Terraform's execution of an operation. It is this behaviour you should think about when deciding which validation to apply.
+
+### Outputs and Variables
+
+[Output postconditions](/terraform/language/expressions/custom-conditions#outputs) and [variable validations](/terraform/language/expressions/custom-conditions#input-variable-validation) both make assertions around inputs and outputs.
+
+In these cases it is often desirable that Terraform should block further execution. For example, it is not useful to warn that an input variable was invalid after an entire Terraform configuration has been applied based on that invalid value. This is the behaviour a check block would apply if an assertion was made against a variable. A validation block against the variable itself would alert before any configuration was applied that the provided input was invalid, and then not create infrastructure based on a flawed premise.
+
+### Resource preconditions and postconditions
+
+The boundary between resource and data source [preconditions and postconditions](/terraform/language/expressions/custom-conditions#preconditions-and-postconditions) and check blocks is more nuanced.
+
+Preconditions are unique amongst the custom conditions in that they execute before a resource change is applied or planned, rather than after. This is discussed in detail in [Choosing Between Preconditions and Postconditions](/terraform/language/expressions/custom-conditions#choosing-between-preconditions-and-postconditions) and the same advice applies when choosing between a precondition and a check block.
```

Comment:
Rewrote to avoid passive voice: 
```suggestion
Preconditions are unique amongst the custom conditions in that they execute _before_ a resource change is applied or planned. [Choosing Between Preconditions and Postconditions](/terraform/language/expressions/custom-conditions#choosing-between-preconditions-and-postconditions) offers guidance on choosing between a precondition and a postcondition, and the same topics also apply to choosing between preconditions and a check block.
```